website/docs/docs/collaborate/model-performance.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Model performance"
+sidebar_label: "Model performance"
+description: "Learn about ."
+---
+
+dbt Explorer provides metadata on dbt Cloud runs for in-depth model performance and quality analysis. This feature assists in reducing infrastructure costs and saving time for data teams by highlighting where to fine-tune projects and deployments &mdash; such as model refactoring or job configuration adjustments.
+
+<LoomVideo id='98f33b3b7a374df0b7c04747eae6ef44' />
+
+:::tip Public preview 
+
+The model performance feature is now available in dbt Explorer! Check it out! 
+:::
+
+## The Performance overview page 
+
+The Performance overview page presents a comprehensive analysis across all project models, helping to pinpoint areas for performance enhancement. It displays the longest-running models, those most frequently executed, and the ones with the highest failure rates during runs/tests. Data can be segmented by environment and job type which can offer insights into:
```

Comment:
I also wonder if this paragraph would benefit from bringing starting with the benefits. Maybe something like:
```suggestion
You can pinpoint areas for performance enhancement by using the Performance overview page. This page presents a comprehensive analysis across all project models and displays the longest-running models, those most frequently executed, and the ones with the highest failure rates during runs/tests. Data can be segmented by environment and job type which can offer insights into:
```